### DISCUSSION TITLE
text attribute changed when calling heading()

### DISCUSSION THREAD

**Discussion by v479038280** [zauberzeug/nicegui#1258] [Q&A]
### Question

I used the heading() method in the websit ,but when I use it ,the defaut attribute with heading() changed

here is the code 

```py
@ui.page('/chart')
def CHART():
    add_costom_menu()

    heading('QTable')
```

I didn't change attribute with heading, here is the default attribute with heading():
```py
def heading(text: str, *, make_menu_entry: bool = True) -> None:
    ui.link_target(create_anchor_name(text))
    ui.html(f'<em>{text}</em>').classes('mt-8 text-3xl font-weight-500 ')
    if make_menu_entry:
        with get_menu():
            ui.label(text).classes('font-bold mt-4')
```

here is the result,the attribute with 'QTable' changed
![微信截图_20230726001602](https://github.com/zauberzeug/nicegui/assets/25096045/56a11a94-c98a-4521-b681-0f796b58b477)

**Comment by v479038280** [comment]
This works fine
```py
@ui.page('/chart')
def CHART():
    add_costom_menu()

    heading('QTable')
    ui.html(f'<em>QTable</em>').classes('mt-8 text-3xl font-weight-500')  
```

![微信截图_20230726003106](https://github.com/zauberzeug/nicegui/assets/25096045/bfcf6d08-484c-4631-8602-633c9bcb6dc1)

**Reply by v479038280** [reply]
There seems two different import .
`from nicegui import globals `

`from nicegui import globals as nicegui_globals
`

**Comment by falkoschindler** [comment]
Sorry, I don't understand your question.
1. What are you trying to do?
2. What have you tried?
3. What happens instead?

If somehow possible, give a [minimal reproducible example](https://en.wikipedia.org/wiki/Minimal_reproducible_example) with proper code formatting and syntax highlighting.

**Reply by v479038280** [reply]
1,  What are you trying to do?   
      I'm trying to resuse the  heading()  method  
2,  What happens instead?  
     for example : I  typed `heading('QTable)` , the result shoud display this  ` ui.html(f'<em>{text}</em>').classes('mt-8 text-3xl font-weight-500 ') `  ,but the result  I see is this (the picture below)
![微信截图_20230727002503](https://github.com/zauberzeug/nicegui/assets/25096045/da947682-04ae-445c-a25c-37e5a66bfef2)
![微信截图_20230727002515](https://github.com/zauberzeug/nicegui/assets/25096045/593aca91-ba51-4410-8375-7964a414ebf9)

 3, What have you tried?
    I show this directly without using this func like this `ui.html(f'<em>'QTable'</em>').classes('mt-8 text-3xl font-weight-500 ')  `

**Reply by falkoschindler** [reply]
Sorry, I really tried to understand your problem. But it's confusing me more and more.
I'll keep the discussion open in case someone else can help.

**Reply by v479038280** [reply]
I've implemented it, just have some doubts and want everyone to see .......

**Reply by v479038280** [reply]
I've implemented it, just have some doubts and discuss it here
